Mathlib/Topology/Defs/Basic.lean

```diff
@@ -110,6 +116,18 @@ def closure (s : Set X) : Set X :=
 def frontier (s : Set X) : Set X :=
   closure s \ interior s
 
+/--
+The coborder is defined as the complement of `closure s \ s`,
+or the union of `s` and the complement of `∂(s)`.
+This is the largest set such that `s` is closed in, and `s` is locally closed if and only if
```

Comment:
I know you just moved the comment, but: I'm stumbling over the grammar of that sentence.
Do you mean "This is the largest set `t` such that `s` is closed in `t`", or "This is the largest set in which `s` is closed"?

Reply:
I added the comment in the first place. To me the the two clauses mean the same thing though. Can you please clarify?

Reply:
I agree with Michael. As written, the sentence doesn't parse.
```suggestion
This is the largest set in which `s` is closed, and `s` is locally closed if and only if
```